test/models/auditing/UserHasPenaltyAuditModelSpec.scala

```diff
@@ -279,13 +279,13 @@ class UserHasPenaltyAuditModelSpec extends SpecBase with LogCapturing {
           expiryReason = None,
           appealInformation = None,
           chargeDueDate = Some(LocalDate.of(2022, 1, 1)),
-          chargeOutstandingAmount = Some(200),
+          chargeOutstandingAmount = Some(0),
```

Comment:
The val name suggests the LSP is unpaid, maybe there could be a new val with a paid LSP?

Reply:
The test it is used suggests there should be paid LSPs "the user has LSPs that are paid and unpaid" so I'll rename the val

Reply:
Done ✅